src/config/session.c

```diff
@@ -118,14 +118,18 @@ read_environment_dir(const char *path_prefix)
 	DIR *envdir = opendir(path);
 
 	if (!envdir) {
-		if (errno != ENOENT) {
+		if (errno == ENOENT) {
+			wlr_log(WLR_INFO,
+				"no %s.d directory found",
+				path_prefix);
+		} else {
```

Comment:
The messaging explicity checks for `ENOENT` because a non-existent directory is generally the "normal" case, so I don't think it merits printing a log message. Please drop this block.